articles/overview/how-to-learn-quantum-computing.md

```diff
@@ -51,7 +51,5 @@ You don’t have to learn this alone, there is a big community of amateurs and e
 * Check out [Q# blog](https://devblogs.microsoft.com/qsharp/) and [Microsoft Quantum Blog](https://cloudblogs.microsoft.com/quantum/) to stay up to date with the latest news and resources about Q#.
 * Check [Q# Community](https://qsharp.community/) and [Awesome Q#](https://project-awesome.org/ebraminio/awesome-qsharp) to look for more resources and material.
 
-## Next steps
+ If you’re looking to teach a course on quantum computing, the [Microsoft Quantum Network](https://info.microsoft.com/LearnMoreAboutMicrosoftQuantumNetwork.html) can help provide curriculum assistance.  
```

Comment:
Would this be reasonable to call out in contributing / community articles as well?

Reply:
sure, but it's not a priority for this PR